@factor/@themes/theme-ultra/src/sidebar.vue

```diff
@@ -62,6 +64,7 @@ export default {
       if (selectedOptions.length > 0) {
         for (const ele of selectedOptions) {
           ele.classList.remove("btn-sidebar-selected");
+          ele.childNodes[1].classList.remove("btn-sidebar-line-selected");
```

Comment:
this should probably be done using a reactive property, always try to avoid manipulating DOM directly

Reply:
You are right, also in terms of performance using vue to manipulate the DOM is probably better, I will change that, thanks!